tests/integration/cmd_create_s2i_test.go

```diff
@@ -0,0 +1,38 @@
+package integration
+
+import (
+	. "github.com/onsi/ginkgo"
+	"github.com/openshift/odo/tests/helper"
+	//. "github.com/onsi/gomega"
+)
+
+var _ = Describe("odo create --s2i command tests", func() {
+	//var oc helper.OcRunner
+	var commonVar helper.CommonVar
+
+	// This is run before every Spec (It)
+	var _ = BeforeEach(func() {
+		//	oc = helper.NewOcRunner("oc")
+		commonVar = helper.CommonBeforeEach()
+		helper.Chdir(commonVar.Context)
+	})
+
+	// Clean up after the test
+	// This is run after every Spec (It)
+	var _ = AfterEach(func() {
+		helper.CommonAfterEach(commonVar)
+	})
+
+	When("creating a component from s2i wildfly", func() {
+
+		BeforeEach(func() {
+			helper.Cmd("odo", "component", "create", "--s2i", "wildfly", "--project", commonVar.Project).ShouldPass()
```

Comment:
Do we need to add that imagestream or does it work as is?

Reply:
I think the imagestream is installed from this script: scripts/configure-installer-tests-cluster.sh (https://github.com/openshift/odo/blob/main/scripts/configure-installer-tests-cluster.sh#L43)